.github/workflows/cicd-2-publish.yaml

```diff
@@ -63,41 +62,36 @@ jobs:
           mkdir -p ./dist
           rm -f ./dist/.gitkeep
           unzip notify-msteams-action-dist.zip -d ./dist
-      - name: "Create release"
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: "Semantic Release"
+        uses: cycjimmy/semantic-release-action@v4.1.0
+        id: semantic
         with:
-          tag_name: ${{ needs.metadata.outputs.version }}
-          release_name: Release ${{ needs.metadata.outputs.version }}
-          body: |
-            Release of ${{ needs.metadata.outputs.version }}
-          draft: false
-          prerelease: false
-      - name: "Upload release asset"
-        uses: actions/upload-release-asset@v1
+          dry_run: ${{github.event_name == 'pull_request'}}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: "${{ steps.create_release.outputs.upload_url }}"
-          asset_path: ./*
-          asset_name: notify-msteams-action-${{ needs.metadata.outputs.version }}.zip
-          asset_content_type: "application/zip"
+      - name: "Output new release details"
+        if: steps.semantic.outputs.new_release_published == 'true'
+        run: |
+          echo ${{ steps.semantic.outputs.new_release_version }}
+          echo ${{ steps.semantic.outputs.new_release_major_version }}
+          echo ${{ steps.semantic.outputs.new_release_minor_version }}
+          echo ${{ steps.semantic.outputs.new_release_patch_version }}
+          export VERSION="${{ steps.semantic.outputs.new_release_version }}"
+          make list-variables
   success:
     name: "Success notification"
     runs-on: ubuntu-latest
     needs: [publish]
     steps:
       - name: "Check prerequisites for notification"
         id: check
-        run: echo "secret_exist=${{ secrets.TEAMS_WEBHOOK_URL != '' }}" >> $GITHUB_OUTPUT
+        run: echo "secret_exist=${{ secrets.TEAMS_NOTIFICATION_WEBHOOK_URL != '' }}" >> $GITHUB_OUTPUT
       - name: "Notify on publishing packages"
         if: steps.check.outputs.secret_exist == 'true'
-        uses: nhs-england-tools/notify-msteams-action@v${{ needs.metadata.outputs.version }}
+        uses: nhs-england-tools/notify-msteams-action@v0.0.4
```

Comment:
Could we set this to @latest here please?

Reply:
I'll change it to user the main branch (latest isn't supported) https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-using-versioned-actions